### DISCUSSION TITLE
Seeed wio e5 LoRa EU868 setting

### DISCUSSION THREAD

**Discussion by dodo124** [danak6jq/Seeed-LoRa-E5#23] [General]
Hi Dana, first off your code is a rellly great.

The  .ioc fille is correct and can be used to generate code. Unfortunatelly you llive in the US, I guess, and you are using it there.

When I try to configure it to EU868 I did not everything right (I guess) because it´s not connecting anywhere?

When using arduino this is somewaht easy and I got some LoRa devices (so I now that the gateway is working) but with the stm32 cube.... Well I mess up somewhat. I can change whateer I want an use whatever method availlable and it´s not responding.

Do you have any idea why this is happening?

Greating Dom

**Comment by danak6jq** [comment]
I do not immediately know what happened there - did you sort it out? This is more of a ST LoRaWAN firmware question perhaps, you might try on the ST community

**Comment by dodo124** [comment]
Hello Dana,

I managed to solve the issue, which was a corrupted file on my disc. After getting another one, it worked flawlessly.

Thanks for your response 😀

Greetings Dom 

Am 25. Juli 2023 19:49:57 MESZ schrieb Dana Myers ***@***.***>:
